src/structures/MessageEmbed.js

```diff
@@ -7,11 +7,19 @@ const Util = require('../util/Util');
  * Represents an embed in a message (image/video preview, rich embed, etc.)
  */
 class MessageEmbed {
-  constructor(data = {}) {
-    this.setup(data);
+  /**
+   * @param {Object} data The data to build the embed from
+   * @param {boolean} skipValidation A flag to skip field validations
```

Comment:
This parameter is for internal use only and should stay undocumented. (Except you can define private parameters, but I don't think you can.)